package.json

```diff
@@ -26,36 +26,9 @@
     "prototype",
     "probuilder"
   ],
-  "samples": [
-    {
-      "displayName": "Universal Render Pipeline Support",
-      "description": "Shaders and materials for the Light Weight Render Pipeline.",
-      "path": "Samples~/UniversalRenderPipeline"
-    },
-    {
-      "displayName": "High Definition Render Pipeline Support",
-      "description": "Shaders and materials for the High Definition Render Pipeline.",
-      "path": "Samples~/HDRP"
-    },
-    {
-      "displayName": "Editor Examples",
-      "description": "Scripts demonstrating how to integrate new functionality into the ProBuilder editor window.",
-      "path": "Samples~/Editor"
-    },
-    {
-      "displayName": "Runtime Examples",
-      "description": "Scripts showing how to create and modify meshes at runtime.",
-      "path": "Samples~/Runtime"
```

Comment:
I added the `samples` property back in the `package.json`. I'm assuming the tests were failing, not because of the `samples` property, but because of the `repository` property, [since `upm.ci` adds that property as well](https://internaldocs.unity.com/upm-ci/faqs/#what-does-upm-ci-modified-existing-properties-in-the-packagejson-manifest-file-mean). (Having that property in the `package.json` created a duplicate when, in turn, `upm.ci` added it.)